src/ts/src/data/cellRenderers.tsx

```diff
@@ -14,10 +18,11 @@ const DEFAULT_RENDERER = (value: JSONPrimitive): JSX.Element => {
 
 const SAMPLE_CUSTOM_RENDERERS: Record<string | number, CellRenderer> = {
   privateId: (value: JSONPrimitive): JSX.Element => {
+    const stringValue = stringGuard(value);
```

Comment:
Did we encounter a bug related to value not casted as a string? Otherwise I'm thinking eventually the value will be a string when it's inserted in the DOM by React?

Reply:
You're right, I got a bit overzealous with adding the type guard